### PR TITLE
Support to serve vLLM on Kubernetes with LWS

### PR DESCRIPTION
**Pull request by kerthcet**
This is on behave of [LWS](https://github.com/kubernetes-sigs/lws) community, we have a minimal example about how to deploy vLLM with lws for multi-host distributed inference. cc @ahg-g @liurupeng

### CODE REVIEW

**Comment by robertgshaw2-neuralmagic** [comment]
@kerthcet looks like the docs build failed

File: docs/source/serving/deploying_with_lws.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+Deploying with LWS
+============================
+
+LeaderWorkerSet (LWS) is low level API aims to address common deploy patterns of AI/ML inference workloads,
+especially designed for multi-host distributed inference needs.
+
+vLLM can be deployed with `LWS <https://github.com/kubernetes-sigs/lws>`_ on Kubernetes for distributed model serving.
+
+Please see `this guide <https://github.com/kubernetes-sigs/lws/tree/main/docs/examples/vllm>`_ for more details on using vLLM with LWS.
```
Comment (robertgshaw2-neuralmagic) [inline docs/source/serving/deploying_with_lws.rst:11]: this link is a 404

File: docs/source/serving/deploying_with_lws.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.. _deploying_with_lws:
+
+Deploying with LWS
+============================
+
+LeaderWorkerSet (LWS) is low level API aims to address common deploy patterns of AI/ML inference workloads,
```
Comment (ahg-g) [inline docs/source/serving/deploying_with_lws.rst:6]:
```suggestion
LeaderWorkerSet (LWS) is a Kubernetes API that aims to address common deploy patterns of AI/ML inference workloads. A major use case it aims to enable is multi-host/multi-node distributed inference.
```

File: docs/source/serving/deploying_with_lws.rst (hunk 1 above)
Comment (ahg-g) [reply docs/source/serving/deploying_with_lws.rst:11]: The correct link is https://github.com/kubernetes-sigs/lws/tree/main/docs/example/vllm

Comment (ahg-g) [inline docs/source/serving/deploying_with_lws.rst:11]:
```suggestion
Please see `this guide <https://github.com/kubernetes-sigs/lws/tree/main/docs/examples/vllm>`_ for more details on deploying vLLM on Kubernetes using LWS.
```

**Review by ahg-g — COMMENTED** [review]
Thanks!

**Comment by simon-mo** [comment]
Please add this to https://github.com/vllm-project/vllm/blob/main/docs/source/serving/integrations.rst?plain=1 to it's included in docs.

File: docs/source/serving/deploying_with_lws.rst (hunk 1 above)
Comment (kerthcet) [reply docs/source/serving/deploying_with_lws.rst:11]: It will take effect after https://github.com/kubernetes-sigs/lws/pull/145, sorry for the confusion.

**Comment by liurupeng** [comment]
+1, thanks!

**Comment by ahg-g** [comment]
Looks good to me, thanks for adding this.

**Review by simon-mo — APPROVED** [review]

**Comment by zhaotyer** [comment]
good work

**Pull request merged**